azure-pipelines.yml

```diff
@@ -107,6 +107,57 @@ jobs:
       TWINE_PASSWORD: $(twine.password)
       TWINE_REPOSITORY_URL: "https://upload.pypi.org/legacy/"
 
+
+# DESCRIPTION: Core API and doc string testing across VTK versions using conda
+- job: LinuxConda
+  pool:
+    vmImage: 'ubuntu-16.04'
+  variables:
+    DISPLAY: ':99.0'
+    PYVISTA_OFF_SCREEN: 'True'
+
+  strategy:
+    matrix:
+      VTKv80:
+        VTK.VERSION: 8.0
+      VTKv81:
+        VTK.VERSION: 8.1
+      VTKv82:
+        VTK.VERSION: 8.2
+
+  steps:
+    - script: |
+        conda install --yes vtk=$(VTK.VERSION)
+        python -c "import pyvista; print(pyvista.Report())"
+      displayName: Install VTK
```

Comment:
This step exists already further down (plus at this point pyvista isn't yet installed).

However there we install vtk after creating the env, which installs all the deps, including vtk. it would probably save a bit of time if we specify the required vtk version when creating the environment (I believe that's possible, not entirely sure) to install the right vtk version from the beginning.

Reply:
Taking a break from this for a bit.  Would you mind implementing this?

Reply:
Sure could do - I just contacted Bane on slack regarding this, as he expressed some desire to fiddle with CI himself there yesterday.

Reply:
NP.  I'll leave it up to Bane's discretion.

Reply:
It was just repeated. I fixed